test/pgtest/affected.pt

```diff
@@ -17,3 +17,40 @@ CommandComplete {"tag":"INSERT 0 2"}
 ReadyForQuery {"status":"I"}
 CommandComplete {"tag":"INSERT 0 10"}
 ReadyForQuery {"status":"I"}
+
+# Our response to UPDATE reflects the number of rows changed by the operation;
+# not the number of rows processed by the operation as in PG
+send
+Query {"query": "UPDATE t SET i = i + 1"}
+----
+
+# The preceding command only changes 3 rows, two of the rows that were 1 are now
+# 2, and one of the rows that was 1 is now 11. The remaining rows remain
+# unchanged from DD's perspective.
+
+until
+ReadyForQuery
+----
+CommandComplete {"tag":"UPDATE 3"}
```

Comment:
I have tried hard to keep our pgtest files identical to postgres output. That's kinda their point, to make sure we implement the protocol and some sql semantics correctly. Tests that purposefully differ from postgres should go in a separate (new) .pt file/directory that is either named or commented so it's obvious that we differ. The way I create compatible pgtest files is to use REWRITE with the `-addr localhost:5432` flag passed to coordtest which generates output from postgres, then I can verify against mz. If we can't rewrite from pg, we should keep those specific parts separate from the pg-compatible tests.